doc/samtools-view.1

```diff
@@ -362,7 +362,16 @@ in octal by beginning with `0' (i.e. /^0[0-7]+/), as a decimal number
 not beginning with '0' or as a comma-separated list of flag names.
 .TP
 .BI "-x " STR ", --remove-tag " STR
-Read tag to exclude from output (repeatable) [null]
+Read tag(s) to exclude from output (repeatable) [null].  This can be a
+single tag or a comma separated list.  Alternatively the option itself
+can be repeated multiple times.
+
+If the list starts with a `^' then it is negated and treated as a
+request to remove all tags except those in \fISTR\fR.
+.TP
```

Comment:
It might be good to specifically document that `-x ^` or `--keep-tag ''` will remove all tags.  It may not be obvious to everyone that this is how you create an empty "keep" list.

It would also be good to say that `--keep-tag` overrides `--remove-tag` if you supply both.